vscode-extensions.json

```diff
@@ -0,0 +1,172 @@
+    },
+    {
+      "version": "0.9.0",
+      "checkout": "master",
+      "repository": "https://github.com/SebastianZaha/vscode-emacs-friendly"
+    },
+    {
+      "version": "0.2.0",
+      "repository": "https://github.com/MicroShed/mp-starter-vscode-ext"
+    },
+    {
+      "version": "1.2.0",
+      "repository": "https://github.com/Azure/vscode-kubernetes-tools"
+    },
+    {
+      "version": "2019.5.18875",
+      "repository": "https://github.com/Microsoft/vscode-python"
+    },
+    {
+      "version": "0.11.4",
+      "repository": "https://github.com/Microsoft/vscode-go"
+    },
+    {
+      "version": "1.33.0",
+      "checkout": "v1.33.0",
+      "repository": "https://github.com/Microsoft/vscode-node-debug2"
+    },
+    {
+      "version": "0.8.0",
+      "checkout": "v0.8.0",
+      "repository": "https://github.com/microsoft/vscode-pull-request-github"
+    },
+    {
+      "version": "0.0.10",
+      "checkout": "v0.0.10-ddc0d3d", 
+      "repository": "https://github.com/pizzafactory-contorno/vscode-libra-move"
+    },
+    {
+      "version": "0.1.0",
+      "checkout": "v0.1.0",
+      "vsixDir": "xtend-vscode-extension/",
+      "repository": "https://github.com/PizzaFactory/xtend-ide-extensions"
+    },
+    {
+      "version": "0.4.0",
+      "checkout": "v0.4.0",
+      "vsixDir": "xtext-vscode-extension/",
+      "repository": "https://github.com/PizzaFactory/xtext-ide-extensions/"
+    },
+    {
+      "version": "0.0.6",
+      "checkout": "v0.0.6", 
+      "repository": "https://github.com/redhat-developer/omnisharp-theia-plugin"
+    },
+    {
+      "version": "0.0.3",
+      "checkout": "v0.0.3", 
+      "repository": "https://github.com/redhat-developer/netcoredbg-theia-plugin"
+    },
+    {
+      "version": "0.0.13",
+      "repository": "https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension"
+    },
+    {
+      "version": "0.57.0",
+      "checkout": "v0.57.0", 
+      "repository": "https://github.com/redhat-developer/vscode-java"
+    },
+    {
+      "version": "1.13.0",
+      "checkout": "v1.13.0", 
+      "repository": "https://github.com/felixfbecker/vscode-php-debug"
+    },
+    {
+      "version": "1.3.11",
+      "checkout": "v1.3.11", 
+      "repository": "https://github.com/bmewburn/vscode-intelephense"
+    },
+    {
+      "version": "0.0.10",
+      "checkout": "v0.0.10", 
+      "repository": "https://github.com/redhat-developer/vscode-project-initializer"
+    },
+    {
+      "version": "1.3.0",
+      "checkout": "v1.3.0", 
+      "repository": "https://github.com/redhat-developer/vscode-quarkus"
+    },
+    {
+      "version": "0.0.23",
+      "checkout": "master", 
+      "repository": "https://github.com/windup/rhamt-vscode-extension"
+    },
+    {
+      "version": "0.0.22",
+      "repository": "https://github.com/camel-tooling/camel-lsp-client-vscode"
+    },
+    {
+      "version": "0.0.13",
```

Comment:
a note to update here or in the other PR https://github.com/eclipse/che-plugin-registry/pull/501 as the version 0.0.14 is on its way

Reply:
Done!